tests/testthat/test-period_dataset.R

```diff
@@ -421,3 +421,44 @@ test_that("create_period_dataset Test 12: DT and DTM columns exist, pulls only o
     keys = c("USUBJID", "APERIOD")
   )
 })
+
+## Test 13: Periods derived even when some variables in dataset_ref are unused ----
+test_that("derive_vars_period Test 13: periods", {
+  expected <- tibble::tribble(
+    ~USUBJID, ~AP01SDT,     ~AP01EDT,
+    "1",      "2021-01-04", "2021-02-06",
+    "2",      "2021-02-02", "2021-03-02"
+  ) %>%
+    mutate(
+      dplyr::across(matches("AP\\d\\d[ES]DT"), ymd)
```

Comment:
```suggestion
      across(matches("AP\\d\\d[ES]DT"), ymd)
```

Reply:
we already import dplry no need to `::` - we do this for tibble as it is a suggest package